### ISSUE TITLE
Can't get translated transcription of video

### ISSUE THREAD

**Issue by LoyalPotato** [jdepoix/youtube-transcript-api#107]
As the title says.
I am using the example provided in the README file, in the translate section.
The video id is -> "Fx_N4GlwdEM" in case you want to try it.

I saw that previously this was an issue ( #30 ) hence why I'm creating a similar issue. 

![image](https://user-images.githubusercontent.com/10759876/113018614-a11bf880-9178-11eb-99c8-9a6d9b707935.png)

**Comment by jdepoix** [comment]
Could you please provide the code you are trying to execute?

**Comment by LoyalPotato** [comment]
```python
from youtube_transcript_api import YouTubeTranscriptApi

# retrieve the available transcripts
transcript_list = YouTubeTranscriptApi.list_transcripts('Fx_N4GlwdEM')

# iterate over all available transcripts
for transcript in transcript_list:

    # the Transcript object provides metadata properties
    print(
        transcript.video_id,
        transcript.language,
        transcript.language_code,
        # whether it has been manually created or generated by YouTube
        transcript.is_generated,
        # whether this transcript can be translated or not
        transcript.is_translatable,
        # a list of languages the transcript can be translated to
        transcript.translation_languages,
    )
    print("\n")

    # fetch the actual transcript data
    print(transcript.fetch())
    print("\n")


    # translating the transcript will return another transcript object
    print(transcript.translate('en').fetch())
    print("\n")

# you can also directly filter for the language you are looking for, using the transcript list
transcript = transcript_list.find_transcript(['de', 'en'])  

# or just filter for manually created transcripts  
transcript = transcript_list.find_manually_created_transcript(['de', 'en'])  

# or automatically generated ones  
transcript = transcript_list.find_generated_transcript(['de', 'en'])

```

**Comment by LoyalPotato** [comment]
Here I was able to get both (error & working) in sequence:
![image](https://user-images.githubusercontent.com/10759876/113023955-1c33dd80-917e-11eb-8e77-bab633384ae8.png)

**Comment by jdepoix** [comment]
What version of python and youtube-transcript-api are you using respectively?

**Comment by LoyalPotato** [comment]
Python: 3.9.2
API: Lastest (0.4.0)

**Comment by jdepoix** [comment]
After not being able to replicate it for a while I discovered that YouTube sometimes returns a page which asks the user to agree to it saving cookies. This is new and happens every now and then. I will have to integrate logic into this module to handle these kind of responses. I will try to fix it as soon as I can and let you know!

**Comment by LoyalPotato** [comment]
Got it, makes sense.
Btw, unrelated, but where can I ask questions about your implementation? Not exactly problems.
Also thanks for looking into it :)

**Comment by jdepoix** [comment]
Fixed in 0.4.1! Thanks for reporting @LoyalPotato! 😊 👍
